src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -21,18 +18,9 @@ public class ErrorNotifier
         private const string NotificationUriSettingName = "AzureWebJobsErrorNotificationUri";
         private readonly string _webNotificationUri;
         private readonly HttpClient _httpClient = new HttpClient();
-        private readonly SendGridAPIClient _sendGrid;
```

Comment:
That definitely sounds like it would be easier to manage - would allow someone to easily see and reapply all the relevant code